topics/transcriptomics/tutorials/ref-based/tutorial.md

```diff
@@ -166,13 +163,42 @@ Sequence quality control is therefore an essential first step in your analysis.
 >    >
 >    {: .question}
 >
+>    As it is tidious to inspect all these reports individually we will combine them with {% tool [MultiQC](toolshed.g2.bx.psu.edu/repos/iuc/multiqc/multiqc/1.11+galaxy0) %}.
+>    However, the current version of the wrapper of **MultiQC** does not accept list of pairs collections.
+>    Thus we will transform the list of pairs to a simple list.
+>
+>    > ### {% icon details %} What does this mean exactly?
+>    > ```text
+>    > Current organisation:
+>    >                                      FastQC results
+>    >                                      /            \
+>    >                   GSM461177_untreat_paired    GSM461180_treat_paired
+>    >                      /           \                   /      \
+>    >                 forward        reverse         forward     reverse
+>    > 
+>    > 
+>    > 
+>    > Desired organisation
+>    >                                       FastQC results
+>    >                                     /     /    \    \
+>    >                                 /       /        \      \
+>    >                             /         /            \        \  
+>    >                         /           /                \          \  
+>    >                     /             /                    \            \
+>    > GSM461177_.._forward   GSM461177_.._reverse   GSM461180_.._forward   GSM461180_.._reverse 
```

Comment:
I hate to ask for more work, but what about an image/chart here, it can be something simple in inkscape or lucidchart or similar.

Reply:
Yes good idea